### PR TITLE
Standardize names

### PR DESCRIPTION
**Pull request by samhunter**
This pull request tries to standardize the naming scheme and also add a bit of branding (hts_) to each of the tools. 
Changes were made to #include lines and cmake files to handle the updated file names. I also tried to make all of the CMake files as generic as possible by replacing hard-coded values with variables whenever possible.

### CODE REVIEW

**Comment by msettles** [comment]
So I built and test on my Mac, also ran my 5M read test dataset, everything looks good to me

File: hts_CutTrim/CMakeLists.txt [hunk 1]
```diff
@@ -20,8 +19,8 @@ add_executable(${PROJECT_NAME} ${sources})
 target_link_libraries(${PROJECT_NAME} ${PROJECT_LINK_LIBS} ${Boost_LIBRARIES})
 
 add_custom_target(valgrind_cut_trim
-  DEPENDS cut_trim
-  COMMAND valgrind ./cut_trim -F -1 ${CMAKE_SOURCE_DIR}/test_suite/fastqFiles/testCase_1X_R1.fastq -2 ${CMAKE_SOURCE_DIR}/test_suite/fastqFiles/testCase_1X_R1.fastq -O
+  DEPENDS ${PROJECT_NAME}
```
Comment (dstreett) [inline hts_CutTrim/CMakeLists.txt]: nioce

File: hts_Overlapper/src/hts_Overlapper.cpp [hunk 2]
```diff
@@ -30,13 +30,20 @@ namespace bi = boost::iostreams;
...
 {
-    const std::string program_name = "overlapper";
+    const std::string program_name = "hts_Overlapper";
     std::string app_description = 
-                       "The overlapper application overlaps paired end reads producing\n";
-
-    app_description += "  a longer overlapped single-end read (lins), or in the case of\n";
-    app_description += "  complete short overlapping (sins) also removes adapters.\n";
-    app_description += "  Reads left not overlapped are lables as 'nins'.\n";
+                       "The hts_Overlapper application attempts to overlap paired end reads\n";
```
Comment (dstreett) [inline hts_Overlapper/src/hts_Overlapper.cpp]: Well done

File: hts_PolyATTrim/src/hts_PolyATTrim.cpp [hunk 3]
```diff
@@ -30,11 +30,12 @@ namespace bi = boost::iostreams;
...
-    const std::string program_name = "polyATtrim";
+    const std::string program_name = "hts_PolyATTrim";
     std::string app_description = 
-                       "PolyAT Trimmer will start on at the end of the reads and works inwards and\n";
-    app_description += "  trim reads when a minimum number <min-trim> of A's or T's with a certain\n";
-    app_description += "  number of mismatches <max_mismatches> is achieved.";
+                       "hts_PolyATTrim trims poly A and T sequences from the 5' end of a read.\n";
+    app_description += "  It start at the 5' end of a read and expands a window towards the 3' end \n";
+    app_description += "  until the max-mismatch parameter is met. If the window is at least\n";
+    app_description += "  <min-trim> bases long, the window will be trimmed from the read.\n";
```
Comment (dstreett) [inline hts_PolyATTrim/src/hts_PolyATTrim.cpp]: Again, very useful, thanks!

**Comment by dstreett** [comment]
Looks good!! Thanks for cleaning things up!

**Comment by msettles** [comment]
Sam, it doesn't look like you did a pull before making the latest changes, need to merge in latest master (n-remover mods) fix whatever the conflics first

**Comment by samhunter** [comment]
Okay, I think I merged master in to catch up with @msettles earlier fixes... Everything still builds and all tests pass.

**Pull request merged**